README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Capstone Project - Web Design Course
+# Webpack Project
```

Comment:
- [ ] Add instructions about how to run your project locally, with all the commands scripts, e.g. `npm install`, `npm run build`, etc.